docs/docs/deploy/resource-requirements.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Resource Requirements
-nav_order: 2
+nav_order: 5
```

Comment:
I had this at the top because generally resources aren’t something you can change after spinning up a server — they have to be configured first. Maybe this page should be edited to clarify?

Reply:
ah understood. I will re-order back to what it was.